docs/user-guide/logging/sdl.md

```diff
@@ -1,14 +1,17 @@
 ---
 assignees:
-- lavalamp
```

Comment:
How about renaming this file to stackdriver.md? sdl.md is a little bit confusing for me.